matchers/toBe.ts

```diff
@@ -0,0 +1,19 @@
+import { expect, fail, pass } from "../newExpect.ts";
+
+declare module "../newExpect.ts" {
+  export interface Matchers<Actual> {
+    toBe: (actual: Actual, expected: unknown) => void;
+  }
+}
+
+expect.register("toBe", (actual, expected) => {
```

Comment:
Demonstrates an implementation of a matcher. This will override the already existing `toBe` function.